### PR TITLE
Update sql-parser.cn.md

### LINKED ISSUE #16043: [OpenForce 2022] Optimize official documents
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Aim

Check the source file below, change `sql` to `SQL` and  keep a space between Chinese and English words.

Source file:
https://github.com/apache/shardingsphere/blob/master/docs/document/content/user-manual/shardingsphere-jdbc/java-api/rules/sql-parser.cn.md

Official document link:
https://shardingsphere.apache.org/document/current/cn/user-manual/shardingsphere-jdbc/java-api/rules/sql-parser/

### PR DESCRIPTION
**Pull request by Swastyy**
Fixes #16043 

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: docs/document/content/user-manual/shardingsphere-jdbc/java-api/rules/sql-parser.cn.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "SQL解析"
+title = "SQL 解析"
```
Comment (zjcnb) [inline docs/document/content/user-manual/shardingsphere-jdbc/java-api/rules/sql-parser.cn.md:2]: The `title` needn't to space.

Comment (Swastyy) [reply docs/document/content/user-manual/shardingsphere-jdbc/java-api/rules/sql-parser.cn.md]: Resolved

**Review by zjcnb — APPROVED** [review]

**Comment by tristaZero** [comment]
Hey @Swastyy I saw you're active recently nice work! Hope you could be the Apache committer of this community! ;-)

**Comment by Swastyy** [comment]
> Hey @Swastyy I saw you're active recently nice work! Hope you could be the Apache committer of this community! ;-)

That is so awesome @tristaZero, I am glad that I can be helpful in the community. Hope to contribute more and more and be a part of the community.

**Pull request merged**